frontend/packages/dev-console/src/components/import/git/GitSection.tsx

```diff
@@ -260,6 +259,18 @@ const GitSection: React.FC<GitSectionProps> = ({
         'devconsole~URL is valid but cannot be reached. If this is a private repository, enter a source Secret in advanced Git options',
       );
     }
+    if (validated === ValidatedOptions.error && buildStrategy === BuildStrategyType.Devfile) {
+      return t('devconsole~No Devfile present, unable to continue');
+    }
+    if (buildStrategy === BuildStrategyType.Source) {
+      return t('devconsole~Repository URL to built and deploy your code from source.');
```

Comment:
```suggestion
      return t('devconsole~Repository URL to build and deploy your code from source.');
```

same for the remaining 2 help texts

Reply:
Thanks. Updated.